### ISSUE TITLE
Make dirble API token configurable and use HTTPS

### ISSUE THREAD

**Issue by lesderid** [notandy/ympd#140]
The API key for Dirble should be configurable, so people can use their own account and the limit isn't reached.

It should also use HTTPS, so if ympd is behind a TLS proxy, the browser won't refuse to load it.

**Comment by tatoosh** [comment]
Any news here for implementation the Dirble API option?

**Comment by lesderid** [comment]
@tatoosh Still waiting on the Dirble dev to enable CORS afaik.

**Comment by tatoosh** [comment]
@lesderid thx. i looked in the sources - can we enable **'Access-Control-Allow-Origin'** in mongoose.h ?

**Comment by lesderid** [comment]
@tatoosh No, it's a change the Dirble dev needs to make on their side, we can't really do anything about it (unless we proxy the API requests, but that might take a bit of work to implement).

**Comment by tatoosh** [comment]
Ok. Should we implement other services like shoutcast?

**Comment by lesderid** [comment]
@tatoosh I'm not sure what you're asking. I suggest opening a separate issue for this.
